Tests/MQTTnet.TestApp.AspNetCore2/Startup.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.FileProviders;
+using MQTTnet;
```

Comment:
Why did you add this? On my machine this is not required. Or do I miss something?